R/makeChartConfig.R

```diff
@@ -23,8 +23,10 @@
 makeChartConfig <- function(dirs, sourceFiles=TRUE){
     # Use the charts settings saved in safetycharts if no path is provided. 
     if(missing(dirs) || is.null(dirs)){
-        #dirs<-paste(.libPaths(),'safetycharts','chartSettings', sep="/")
-        dirs<-paste(.libPaths(),'safetycharts','config', sep="/")
+        for(lib in .libPaths()){
+            dirs<-paste(lib,'safetyCharts','config', sep="/")
+            if(file.exists(dirs)) break               
```

Comment:
So basically choose the first libPath.  This seems to do the trick.

Reply:
Good point @xni7, it's not immediately obvious to me how `safetyCharts` fits into the workflow when using a custom set of chart config files.  It seems the workflow functions are pulled both from the location of the chart config files as well as from the `safetyCharts` package.

Reply:
Yeah, you definitely *can* run `safetyGraphics` without `safetyCharts` if you supply all your own renderers and config files, but I'm not sure how often that will actually happen. 

For this chunk of code, `safetyCharts` *is* required unless the `dirs` option is set. So I think a warning is totally reasonable. I'll add something now and then merge.